docker/README.md

```diff
@@ -1,9 +1,27 @@
+# Launch New Instance
+
+1. Use "Launch instance from template"
+Use the `alchemiscale-template` to lauch a new EC2 instance.
+The only setting that will need modification is under "Key pair (login)".
+Select "Proceed without a key pair" (SSH keys have already be added to the image).
+
+2. Setup DNS
+Copy the public IP from the newly launched instance.
+Use that IP address to create an A record with desired hostname.
```

Comment:
Would be useful to have some instructions on how to do this for a rank AWS beginner like me.

Reply:
These instructions assume a user deploying `alchemiscale` to their AWS environment has an `alchemiscale-template` defined. This is not generally true, and the `docker-compose`-based instructions are a better fit for generality.

I think instructions like these may make sense for an organization's internal docs on how to re-deploy a host from an existing template, but they don't belong in the `alchemiscale` codebase repo itself.